.github/workflows/ci.yml

```diff
@@ -332,8 +332,8 @@ jobs:
       - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.2
       - name: "Init"
         run: bash ./scripts/ci/init.sh
-      - name: "Run PostgreSQL 14.9 database"
-        run: docker-compose -f ./scripts/ci/docker-compose/docker-compose-db.yaml --profile postgres up -d
+      - name: "Run PostgreSQL 13.7 database"
```

Comment:
We shouldn't change the version.

Reply:
Tyvm! Must have been some automated conflict resolution issues.